app/Services/Campaign/ExportService.php

```diff
@@ -142,27 +144,19 @@ protected function gallery(): self
         return $this;
     }
 
-    protected function processFolder($image, string $path): string
+    protected function processImage($image): self
     {
-        $route = $path . '/' . $image->name;
-
-        if ($image->imageFolder) {
-            $route = $route . '/';
-            $route = $this->processFolder($image->imageFolder, $route);
-        } else {
-            $route = $route . '.' . $image->ext;
+        if(!$this->assets) {
```

Comment:
Return early to avoid code being nested

Reply:
Its set the same way as the already existing process function, I dont see a way of returning earlier.